src/Components/Counter.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+const title = 'Counter';
+export class Counter extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            counter: parseInt(props.initialValue)
```

Comment:
Also, thinking about flexibility might be good to have a default value if is not set
`props.initialValue || 0`

Reply:
You are right.